staging/src/k8s.io/kubelet/pkg/apis/deviceplugin/v1alpha/api.pb.go

```diff
@@ -607,7 +609,7 @@ func NewRegistrationClient(cc *grpc.ClientConn) RegistrationClient {
 
 func (c *registrationClient) Register(ctx context.Context, in *RegisterRequest, opts ...grpc.CallOption) (*Empty, error) {
 	out := new(Empty)
-	err := c.cc.Invoke(ctx, "/deviceplugin.Registration/Register", in, out, opts...)
+	err := c.cc.Invoke(ctx, "/v1alpha.Registration/Register", in, out, opts...)
```

Comment:
same question about whether this means the endpoint or wire format/content is changing